README.md

```diff
@@ -1,9 +0,0 @@
-# FortiSOAR Solution Pack: Automated Employee Onboarding
-
-[![Solution Pack Video](https://img.youtube.com/vi/ytVDEOY432A/0.jpg)](https://www.youtube.com/embed/ytVDEOY432A)
```

Comment:
need to correct the video, starting bits are not accurate . the installation process will be different in the solution pack case.

Reply:
Ankit, we will need to ask original submitter of this PR for this change.

Reply:
@ftnt-cse, can we make video more presentable?